habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -315,69 +225,43 @@ def _init_train(self, resume_state=None):
         ):
             os.makedirs(self.config.habitat_baselines.checkpoint_folder)
 
-        self._setup_actor_critic_agent(ppo_cfg)
-        if resume_state is not None:
-            self.agent.load_state_dict(resume_state["state_dict"])
-            self.agent.optimizer.load_state_dict(resume_state["optim_state"])
-        if self._is_distributed:
-            self.agent.init_distributed(find_unused_params=False)  # type: ignore
+        logger.add_filehandler(self.config.habitat_baselines.log_file)
```

Comment:
Is this the best place to put this logger change? I would expect it to be better at the very beginning. Like in run.py.